Makefile

```diff
@@ -1,6 +1,8 @@
+include .functional-tests.env
```

Comment:
Why do we need to do this?.

Reply:
It isn't needed, I forgot to remove it, I've removed it now